autoload/ale/completion.vim

```diff
@@ -433,7 +429,7 @@ function! ale#completion#ParseLSPCompletions(response) abort
 
         call add(l:results, {
         \   'word': l:word,
-        \   'kind': l:kind,
+        \   'kind': ale#completion#GetCompletionSymbols(get(l:item, 'kind', '')),
```

Comment:
`l:item.kind` is  a number, as defined above, and there are a few more in the spec now too. https://microsoft.github.io/language-server-protocol/specification

Maybe we should map those numbers to strings that are easier for users to configure the new setting with.